controllers/orders/change-status/start-cooking/index.js

```diff
@@ -0,0 +1,32 @@
+import { orderStatus } from '#constants/orderStatus.js';
+import { changeOrderStatus } from '#controllers/orders/helpers.js';
+import { isValidParameterId, ctrlWrapper } from '#middlewares/index.js';
+import Order from '#models/order/index.js';
+
+const controller = async (req, res) => {
+  const { orderId } = req.params;
+
+  // mock
+  const chefId = '6557219bccbbbbc3695bc8b2';
+
+  const order = await Order.findOne({ _id: orderId }).exec();
+
+  order.status = changeOrderStatus(order, {
+    currentStatuses: [orderStatus.ACCEPTED, orderStatus.PENDING],
```

Comment:
if this property identifies from which statuses setting `nextStatus` is forbidden, it's better to rename it

Reply:
No, on the contrary, it is just a field that shows from which statuses it is allowed to change the status

Reply:
Changed to allowedStatuses